examples/mxnet/gcmc/data.py

```diff
@@ -0,0 +1,471 @@
+import numpy as np
```

Comment:
Now that we have heterogeneous API available, I guess we need to have MovieLens, ACM and other datasets in `dgl.data` in the next release.

Reply:
cc @VoVAllen. My question is how should we store the data? The current script directly loads the raw data (stored in tsv) and preprocesses it by pandas. Since the whole loading process is not that long, I feel we could keep the current practice and add some logic to download the raw data from the official site.

Reply:
I think we should use raw data if processing time is acceptable. Since it's an important part of the model, as well as the final performance. We can provide preprocessed file if needed in the future.